.circleci/config.yml

```diff
@@ -79,8 +79,14 @@ jobs:
           name: Build Pub/Sub (C#)
           command: artman --api=pubsub --language=csharp --publish=noop
       - run:
+          name: Build Pub/Sub (C#/gRPC)
+          command: artman --api=pubsub --language=csharp --pipeline=GrpcClientPipeline --publish=noop
```

Comment:
Can we make C# work like the other languages where the GAPIC pipeline includes the GRPC pipeline? (This just requires removing my two-line `if` block that circumvented this behavior.)